### PR TITLE
Add Automation for Tweets

### PR DESCRIPTION
**Pull request by bnb**
This PR adds automation for tweeting both the build tweet and the hiring tweet. It also adds tooling that can be reused to automatically write other tweets with minimal additional work.

### CODE REVIEW

**Review by gr2m — COMMENTED** [review]
very cool!

**Review by joesepi — APPROVED** [review]
LGTM. Thanks again!

I paired on this, but next day review: everything checks out. :)

File: .github/workflows/build-tweet.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - run: "./scripts/recurring/build.js" # run the build tweet tweet builder
+      - uses: gr2m/create-or-update-pull-request-action@v1.x # create a PR or update the Action's existing PR
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (gr2m) [inline .github/workflows/build-tweet.yml:21]: note that if you create a pull request using `secrets.GITHUB_TOKEN`, no other actions will be triggered by the `pull_request` event. You will have to pass a personal access token instead. I'd reccomend to use a bot account or create one for this purpose only

File: .github/workflows/hiring-tweet.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - run: "./scripts/recurring/hiring.js" # run the hiring tweet tweet builder
+      - uses: gr2m/create-or-update-pull-request-action@v1.x # create a PR or update the Action's existing PR
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (gr2m) [inline .github/workflows/hiring-tweet.yml:21]: same as above: if you create a pull request using `secrets.GITHUB_TOKEN`, no other actions will be triggered by the `pull_request` event. You will have to pass a personal access token instead. I'd reccomend to use a bot account or create one for this purpose only

**Review by gr2m — COMMENTED** [review]
sorry I didn't look for this before

**Pull request merged**